pkg/features/kube_features.go

```diff
@@ -480,6 +480,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	ProcMountType:                               {Default: false, PreRelease: utilfeature.Alpha},
 	TTLAfterFinished:                            {Default: false, PreRelease: utilfeature.Alpha},
 	KubeletPodResources:                         {Default: false, PreRelease: utilfeature.Alpha},
+	WindowsGMSA:                                 {Default: false, PreRelease: utilfeature.Alpha},
```

Comment:
i do not think i reviewed the prior pr, but i am happy the gate was added.

do we know its covering all the impacted areas?

Reply:
@derekwaynecarr : the feature gate was already in the prior PR, and protecting all the impacted areas. I "just" had forgotten to also add it to this list, much to my shame.

Reply:
nothing in this pr appears to be gating on this feature gate.  can we ensure gating happens by adding the feature gate with the reader in a separate pr?

Reply:
I'm sorry, I'm not sure I understand what you're suggesting?

Reply:
For reference, this was the earlier PR where the feature gating for `WindowsGMSA` is done: https://github.com/kubernetes/kubernetes/pull/73726/files#diff-4d8e81d13b02ca9a4b8919ef8cf8b6f9R40

Reply:
Sorry for not catching that in the previous review.

Reply:
@yujuhong : hardly your fault :)